### PR TITLE
suggestions for metadata.qmd text

### PR DESCRIPTION
**Pull request by imartorelli**
- comments added on text file for suggestions on explaining concepts

### CODE REVIEW

**Review by Karvovskaya — APPROVED** [review]
thank for the thoughtful comments and improvements, Irene!  I think it would be good to clarify the "controlled vocabularies & ontologies" part!

File: topics/metadata.qmd [hunk 1]
```diff
@@ -30,14 +30,21 @@ Examples of controlled vocabularies are:
...
 - [Medical Subject HEadings (MeSH)](https://www.nlm.nih.gov/mesh/meshhome.html)
+- [The Environment Ontology (EnvO)](https://sites.google.com/site/environmentontology/Browse-EnvO?authuser=0)
 
-Many examples of vocabularies and classification systems can be found at the [FAIRsharing.org website](https://fairsharing.org/). It has  a large list for multiple disciplines. If you are working on new concepts or new ideas and are using or creating your own ontology/terminology, be sure to include them as part of the metadata documentation in your dataset (for example as part of your [codebook](../data-documentation.qmd#codebook).
+Many examples of vocabularies and classification systems can be found at the [FAIRsharing.org website](https://fairsharing.org/). It has  a large list for multiple disciplines. If you are working on new concepts or new ideas and are using or creating your own ontology/terminology, be sure to include them as part of the metadata documentation in your dataset (for example as part of your [codebook](../data-documentation.qmd#codebook)).
```
Comment (chartgerink) [inline topics/metadata.qmd:35]:
```suggestion
Many examples of vocabularies and classification systems can be found at the [FAIRsharing.org website](https://fairsharing.org/). It has  a large list for multiple disciplines. If you are working on new concepts or new ideas and are using or creating your own ontology/terminology, be sure to include them as part of the metadata documentation in your dataset (for example as part of your [codebook](../topics/data-documentation.qmd#codebook)).
```

Fixing the link 😊

File: topics/metadata.qmd [hunk 2]
```diff
@@ -30,14 +30,21 @@ Examples of controlled vocabularies are:
...
+[comment]: # (codebook url missing)
+[comment]: # (Something to think about..I am afraid I am missing in this section the connection and the use of the terms "controlled vocabularies & ontologies". The title of this section is Controlled Vocabularies and Classification. For a researcher who is not familiar with these terms, I am afraid they do not know that a controlled term can be from a term list, while an ontology is a structured classified term. Thus it may be sometimes confusing to read only the term "controlled vocabulary", at some point in the text only read the term "ontology", or read the sentence "controlled vocabularies and ontologies". Thus the question a researcher might have is "are these two terms interchangeable?")
+
+[comment]: # (perhaps worth to mention the OntoPortal as a resource in this page?)
```
Comment (chartgerink) [inline topics/metadata.qmd:39]:
```suggestion
```

Documented suggestion in #285.

File: topics/metadata.qmd [hunk 3]
```diff
@@ -30,14 +30,21 @@ Examples of controlled vocabularies are:
...
 
 ## Metadata levels
 
 Finally a distinction can be made on the level of description. Metadata can be about the data as a whole or about part of the data. It can depend on the research domain and the tools that are used on how many levels the data can be described. In repositories like Yoda and DataverseNL it is common practice to only create structured metadata on the level of the data as a whole. The Consortium of European Social Science Data Archives (CESSDA) explains this distinction for several types of data in their [Data Management Expert Guide](https://dmeg.cessda.eu/Data-Management-Expert-Guide/2.-Organise-Document/Documentation-and-metadata).
 
-![Flowchart indicating a project with a Folder A and Folder B, where Folder A has File 1 and File 2. The project, Folder A, and File 1, have linked metadata to them.](../public/metadata_levels.png)
+![Flowchart indicating a project with a Folder a and Folder b, where Folder a has File 1 and File 2. The project, Folder a, and File 1, have linked metadata to them.](../public/metadata_levels.png)
+
+[comment]: # (an example of how a metadata file would look like?)
```
Comment (chartgerink) [inline topics/metadata.qmd:47]:
```suggestion
![Flowchart indicating a project with a Folder a and Folder b, where Folder a has File 1 and File 2. The project, Folder a, and File 1, have linked metadata to them.](../public/metadata_levels.png)
```

I need to remove the comment in order to merge the improvements - not an assessment of the comment itself! Please feel free to raise this in an issue as well if it should be discussed more. 😊

**Review by chartgerink — APPROVED** [review]

**Pull request merged**